testing/test_setuponly.py

```diff
@@ -292,3 +294,19 @@ def test_arg(arg):
         ]
     )
     assert result.ret == ExitCode.INTERRUPTED
+
+
+def test_parametrize_no_comparing_bytearray_error(testdir):
```

Comment:
As said above, the issue is not comparison so this test name should be adjusted.

Would also be good to add a comment `# Issue #7126` to help anyone who stumbles on this test to see the context.